test/unit/breadcrumbs.ts

```diff
@@ -112,14 +110,14 @@ suite('Breadcrumbs', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveA
     it('should set fields', () => {
       const state = { a: 'b' };
       const navigations = [{ selected: [1], field: 'c' }, { selected: [2, 3], field: 'd' }, { selected: [] }];
-      const selectNavigations = stub(Selectors, 'navigations').returns(navigations);
+      const select = breadcrumbs.select = <any>spy(() => navigations);
```

Comment:
should already be the `select` reference from `beforeEach()`

Reply:
The spy needs to return something different though. Is it enough to set `breadcrumbs.select = <any>spy(() => navigations)` ?